scripts/deploy.py

```diff
@@ -261,9 +262,32 @@ def _execute_deployment():
         if build_process.returncode > 0:
             raise Exception('Build failed.')
 
+        # Update indexes, then prompt for a check that they are all serving
+        # before continuing with the deployment.
+        assert os.path.isfile(INDEX_YAML_PATH)
+        subprocess.check_output([
+            common.GCLOUD_PATH, '--quiet', 'datastore', 'indexes', 'create',
+            INDEX_YAML_PATH, '--project=%s' % APP_NAME])
+        datastore_indexes_url = (
+            'https://console.cloud.google.com/datastore/indexes?project=%s' %
+            APP_NAME)
+        common.open_new_tab_in_browser_if_possible(datastore_indexes_url)
+        while True:
+            print '******************************************************'
+            print (
+                'PLEASE CONFIRM: are all datastore indexes serving? See %s '
+                '(y/n)' % datastore_indexes_url)
+            answer = raw_input().lower()
+            if answer in ['y', 'ye', 'yes']:
+                break
+            elif answer:
+                raise Exception(
```

Comment:
If it exits here, the script would need to rebuild (when rerun)? But I think logically this check can only happen after the build (I assume indexes are updated on build... No idea when they are changed).

Reply:
Indexes just rely on index.yaml, which isn't touched by the build.

Reply:
So should we do this check before building? (Just to save some time)

Reply:
Ah, I see, I think I misunderstood the point of your earlier comment.

I put it here for future-proofing, actually. It's not touched by the build now but I don't know whether it will in the future, so I wanted to do all the "talk to server" stuff only once we confirm that we have a finalized build. But I think you make a good point. I'll move it before the build in an upcoming commit.

Reply:
Done!